client/dive-common/components/Viewer.vue

```diff
@@ -507,12 +523,15 @@ export default defineComponent({
           </v-alert>
           <v-progress-circular
             v-else
-            indeterminate
+            :indeterminate="progress.progress < 0"
+            :value="Math.round(progress.progress / progress.total * 100)"
             size="100"
             width="15"
             color="light-blue"
+            class="main-progress-linear"
           >
-            Loading
+            <span v-if="progress.progress < 0">Loading</span>
```

Comment:
feel free to ignore:  This line for some reason is hard for my brain to process quickly.  I think it's the ` progress < 0">Loading</span>`  Think we could force this in the formatter onto a new line?

Reply:
Also sure.